hw/ip/flash_ctrl/dv/flash_ctrl_base_sim_cfg.hjson

```diff
@@ -210,6 +210,54 @@
       run_opts: ["+scb_otf_en=1"]
       reseed: 10
     }
+    {
+      name: flash_ctrl_wo_ecc
+      uvm_test_seq: flash_ctrl_wo_vseq
+      run_opts: ["+scb_otf_en=1", "+ecc_mode=1"]
+      reseed: 5
+    }
+    {
```

Comment:
quick question, what's the reason these tests only have 1 reseed? is it because the tests have no randomization?

Reply:
yes, write / read word sweep basically, do operation (write / read) starting from 1 4byte word and increment by one 4byte word each round all the way up to 16 4 byte word.

Only randomization here is starting offset, (start from address 0 or address 4).
So I set to 1 iteration.